core/common/src/main/java/com/alipay/sofa/rpc/common/utils/CommonUtils.java

```diff
@@ -275,4 +272,43 @@ public static String join(Collection collection, String separator) {
         }
         return sb.length() > 0 ? sb.substring(0, sb.length() - separator.length()) : StringUtils.EMPTY;
     }
+
+    /**
+     * 将String数组转化为map
+     *
+     * @param pairs  目标数组
+     * @return HashMap
+     */
+    public static Map<String, String> toStringMap(String... pairs) {
+        Map<String, String> parameters = new HashMap<String, String>();
+        if (pairs.length > 0) {
+            if (pairs.length % 2 != 0) {
+                throw new IllegalArgumentException("pairs must be even.");
+            }
+            for (int i = 0; i < pairs.length; i = i + 2) {
+                parameters.put(pairs[i], pairs[i + 1]);
+            }
+        }
+        return parameters;
+    }
+
+    /**
+     * 比较集合是否一致，忽略顺序
+     *
+     * @param newCollection  新集合
+     * @param oldCollection 旧集合
+     * @return 是否一致
+     */
+    public static boolean isSameCollection(Collection<?> newCollection, Collection<?> oldCollection) {
```

Comment:
The method is equivalent to `public static <T> boolean listEquals(List<T> left, List<T> right) `